test/twobinomial.jl

```diff
@@ -0,0 +1,88 @@
+using HypothesisTests, Test
+
+@testset "Two Binomial" begin
```

Comment:
Have you checked all the values hardcoded here against other publications or software? Given the number of methods and special cases for each of them, I imagine these tests don't cover all branches, do they?